contract/grpc/proto/edge-cluster.proto

```diff
@@ -117,6 +120,9 @@ message UpdateEdgeClusterResponse {
 message DeleteEdgeClusterRequest {
   // The unique edge cluster identifier
   string edgeClusterID = 1;
+
+  // The edge cluster object contains the edge cluster details to delete
+  EdgeCluster edgeCluster = 2;
```

Comment:
You don't need this, get the edge cluster ID only and use the repository layer to retreive the TenantID, so you can find out the details of the edge cluster